hack/local-up-cluster.sh

```diff
@@ -704,7 +706,11 @@ function start_kubelet {
     mkdir -p "/var/lib/kubelet" &>/dev/null || sudo mkdir -p "/var/lib/kubelet"
     # Enable dns
     if [[ "${ENABLE_CLUSTER_DNS}" = true ]]; then
```

Comment:
The usage of `==` vs `=` in this file is confusing but I see we do this elsewhere in this file.